docs/docs/projects/README.md

```diff
@@ -77,8 +79,72 @@ At this time it's not possible to move workflows out of GitHub Synchronized Proj
 
 :::
 
+## Access permissions
+
+### Understanding who has access to which project
```

Comment:
```suggestion
```

Consider removing this heading, typically all headers have content underneath. I think the content alone does the job.